styles.css

```diff
@@ -657,8 +658,216 @@
+  .languages {
+    position: static;
+    margin-right: 8px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: white;
+    background: #26815b;
+    border-radius: 4px;
+    border: none;
+    height: 32px;
+  }
+
+  .see-project {
+    background: #26815b;
+    border-radius: 4px;
+    width: 122px;
+    height: 48px;
+    top: 12px;
+    left: 12px;
+    font-family: 'Inter', sans-serif;
+    color: white;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    margin-top: 38px;
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    border: none;
+  }
+
+  .see-project:hover {
+    background-color: #36b37e;
+    color: #fff;
+  }
+
+  .overlay {
+    display: none;
+    position: fixed;
+    z-index: 1;
+    left: 0;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    overflow: auto;
+    background-color: rgba(0, 0, 0, 0.8);
+  }
+
+  .pop-up-modal {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    margin-top: 165px;
+    transform: translate(-50%, -50%);
+    background-color: white;
+    padding: 0 1.5rem;
+    padding-bottom: 1.5rem;
+    border-radius: 0.5rem;
+  }
+
+  .close-btn-cont {
+    display: flex;
+    justify-content: flex-end;
+    height: 20px;
+    width: 830px;
+    margin: 100px 20px 0 0;
+  }
+
+  .img-cont-popup {
+    display: block;
+    text-align: center;
+    padding-top: 10px;
+  }
+
+  #img {
+    transform: scale(0.95);
+  }
+
+  .popped-window {
+    display: flex;
+    justify-content: space-between;
+    margin: 0 30px 0 30px;
+  }
+
+  #name {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 32px;
+    line-height: 44px;
+    color: #172b4d;
+  }
+
+  .flex-btn {
+    display: flex;
+  }
+
+  .live {
+    background: #26815b;
+    border-radius: 4px;
+    width: 122px;
+    height: 48px;
+    font-family: 'Inter', sans-serif;
+    color: white;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    margin: 20px 5px 20px 5px;
+    border: none;
+    text-decoration: none;
+  }
+
+  .buttons-tech {
+    margin: 0 0 0 25px;
+  }
+
+  .technologies-buttons {
+    margin: 0 4px 0 4px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #344563;
+    background: #ebecf0;
+    border-radius: 8px;
+    border: none;
+    height: 32px;
+    padding: 8px 8px 0 8px;
+  }
+
+  .lorem {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 20px;
+    line-height: 28px;
+    margin: 15px 25px 0 25px;
+    color: #172b4d;
+  }
+
+  .flex-btn-mob {
+    display: none;
+  }
+
+  .live-mob {
+    display: none;
+  }
+
+  .ul-container {
+    padding-inline-start: 0;
+    display: flex;
+    list-style: none;
+  }

```

Comment:
- [ ] Great job, it would be best to center these items vertically so that it looks great and we stick to the design.

### Your Implementation
![image](https://user-images.githubusercontent.com/59416136/167944646-8eb0bc76-13fc-4151-bc03-2160460e41fe.png)

### Figma's Design
![image](https://user-images.githubusercontent.com/59416136/167944706-a5eb2b26-56f5-40cd-be79-39f9d7ed7611.png)

Reply:
Thank you for that. I had done according to the figma design where the button were left aligned. Do you think there is need to change that?

Figma's design

![image](https://user-images.githubusercontent.com/59999191/167953663-44d09cb2-d095-4cd6-bc30-5770cc91b9df.png)

My design

![image](https://user-images.githubusercontent.com/59999191/167953509-2320bb94-3a00-410d-b4ac-aa94f6719160.png)

Reply:
@Wes-Isaac